<?xml version="1.0"?>
<launch>

  <rosparam command="load" file="$(find denmpc)/config/denmpc.yaml" />

  <node name="mpc_node" pkg="denmpc" type="mpc_node" output="screen"/>

</launch>
